core/transaction.go

```diff
@@ -320,23 +312,23 @@ func deployAccountTransactionHash(d *DeployAccountTransaction, n utils.Network)
 			d.Nonce,
 		), nil
 	}
-	return nil, ErrInvalidTransactionVersion{d, d.Version.Text(felt.Base10)}
+	return nil, errInvalidTransactionVersion(d, d.Version)
 }
 
-type ErrCantVerifyTransactionHash struct {
+type CantVerifyTransactionHashError struct {
```

Comment:
@IronGauntlets thought we wanted to follow the convention `Err...`? Uber Guidelines also suggested the same https://github.com/uber-go/guide/blob/master/style.md#error-naming

Reply:
yeah you are right I forgot to make it private